test/src/e2e_vm_tests/test_programs/should_pass/language/struct_destructuring/test.toml

```diff
@@ -0,0 +1,3 @@
+category = "run"
+expected_result = { action = "return", value = 42 }
+validate_abi = true
```

Comment:
as this isn't a contract, do we need to validate the abi?

Reply:
Good catch!  I'm not sure, there are other test cases in the same folder such as tuple_access that are scripts and validate the abi.

Reply:
Hm, I don't think it is necessary. Cc @otrho for consensus on that.

Reply:
Time for an RFC!

Reply:
We've bee validating the ABI for all passing tests including scripts because scripts emit a JSON ABI now. Not sure how important that is though.

Reply:
BTW, all I did was refactor the test harness.  This doesn't make me the authority on how or what to test. :wink: